From 1fe702291728a81388305a797329e5ac4956b8c4 Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Wed, 21 Jan 2015 15:26:13 +0100
Subject: [PATCH] util: Add some missing hidden_file() suffixes

dpkg itself also uses *.dpkg-dist, while .dpkg-{bak,backup,remove} are being
used by dpkg-maintscript-helper.

(cherry picked from commit c7088e4999f2e5dd33259948c806f4e2706e77ce)
---
 src/shared/util.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/shared/util.c b/src/shared/util.c
index 688aa8bc13..6f22f2b04e 100644
--- a/src/shared/util.c
+++ b/src/shared/util.c
@@ -1441,6 +1441,10 @@ _pure_ static bool ignore_file_allow_backup(const char *filename) {
                 endswith(filename, ".dpkg-old") ||
                 endswith(filename, ".dpkg-new") ||
                 endswith(filename, ".dpkg-tmp") ||
+                endswith(filename, ".dpkg-dist") ||
+                endswith(filename, ".dpkg-bak") ||
+                endswith(filename, ".dpkg-backup") ||
+                endswith(filename, ".dpkg-remove") ||
                 endswith(filename, ".swp");
 }
 
